enclave/crypto/cert.c

```diff
@@ -1181,3 +1181,46 @@ oe_result_t oe_gen_custom_x509_cert(
 
     return result;
 }
+
+oe_result_t oe_cert_get_validity_dates(
+    const oe_cert_t* cert,
+    oe_datetime_t* not_before,
+    oe_datetime_t* not_after)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    const Cert* impl = (const Cert*)cert;
+
+    if (not_before)
+        memset(not_before, 0, sizeof(oe_datetime_t));
```

Comment:
Would combining his memset statement into the next if (not_before) block make the code cleaner?
The same comment applies to not_after

Reply:
Agree.